packages/cli/helpers/build.ts

```diff
@@ -48,13 +48,51 @@ const cliLifecyclePlugin: esbuild.Plugin = {
   },
 }
 
+/**
+ * Loading WASM modules in esbuild seems to be quite tricky, and none of the plugins/tutorials available online
+ * seem to cut it.
+ * Currently, we import WASM modules based on the following assumptions:
+ * - they're published on npm with a name like `@prisma/<package-name>-wasm`
+ * - they're installed in the `internals` package, where they're left unbundled
+ */
```

Comment:
Ideally, we'll reach a point where those assumptions can be relieved, so that any WASM module can be imported without breaking at runtime